.tributors

```diff
@@ -63,7 +63,8 @@
     "adelavega": {
         "name": "adelavega",
         "blog": "https://github.com/adelavega",
-        "bio": "Postdoc in the Psychoinformatics Lab"
+        "bio": "Postdoc in the Psychoinformatics Lab",
+        "email": "aleph4@gmail.com"
```

Comment:
@adelavega no objections to have your email exposed here?

Reply:
```suggestion
        "email": "delavega@utexas.edu"
```